scripts/esbuild-plugins/utils.js

```diff
@@ -58,7 +58,7 @@ exports.createAliasReplacer = (aliases) => {
 
     return source.replace(re, (match, capture) => {
       const resolvedPathSegment = aliases[capture]
-      const replacement = join(relativeDirPath, resolvedPathSegment)
+      const replacement = join(relativeDirPath, resolvedPathSegment).replace(/\\/g, '/')
```

Comment:
Woah! Isn't `join` supposed to be platform agnostic? (The entire point of having a `join` function?)

Reply:
`join` accepts paths containing either `/` or `\\`, but returns Windows paths when on Windows, which we don't want here since that whould change the `replacement` value

Reply:
Oh ok, @snowteamer, could you add a comment explaining this then?

Reply:
@greg Done :)